test/unit/attribute_test.rb

```diff
@@ -266,14 +266,13 @@ def create_attribute(model, name)
   end
 
   test "limit should return nil for oddball column types that misuse the limit attribute" do
-    create_model "Business", :location => :integer
-    attribute = create_attribute(Business, "location")
-    attribute.column.class_eval do
-      define_method :limit do
-        # https://github.com/voormedia/rails-erd/issues/21
-        { :srid => 4326, :type => "point", :geographic => true }
-      end
+    create_model "Business", :location => :integer do
+    define_singleton_method :limit do
+      # https://github.com/voormedia/rails-erd/issues/21
+      { :srid => 4326, :type => "point", :geographic => true }
     end
+  end
```

Comment:
I feel like we've got an indentation issue or missing `end` here?

Reply:
Yep, looks like the indentation was wrong, should be fixed now!

Reply:
Yep.. I'll fix it in a follow-up MR